timerapplet/src/timerapplet.c

```diff
@@ -162,7 +162,9 @@ timer_callback (TimerApplet *applet)
             gtk_widget_set_tooltip_text (GTK_WIDGET (applet->label), name);
             gtk_widget_hide (GTK_WIDGET (applet->pause_image));
             atk_object_set_name (atk_obj, label);
-            atk_object_set_description (atk_obj, tooltip);
+
```

Comment:
Excess blank lines

Reply:
I can get those out now, in another PR have to kill a build warning from earlier work

Reply:
Also found the code flow was different than I thought (`tooltip `never takes the time remaining) but the fix is the same

Reply:
Done, excess blank lines are gone